python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -422,7 +441,7 @@ async def _process_function_call(
         arguments: "KernelArguments",
         function_call_count: int,
         request_index: int,
-        function_call_behavior: FunctionCallBehavior,
```

Comment:
Per @TaoChenOSU's point, this `_process_function_call` method should probably be moved to a common location since other function calling code will need to leverage it.